oioswift/proxy/controllers/obj.py

```diff
@@ -410,7 +412,8 @@ def _link_object(self, req):
             storage.object_fastcopy(
                 from_account, container, obj,
                 self.account_name, self.container_name, self.object_name,
-                headers=oio_headers)
+                headers=oio_headers, metadata=metadata,
+                properties_directive='REPLACE')
```

Comment:
I think we should use `properties=metadata` since `properties` is the new keyword, and both were ignored in old versions.

Reply:
Done